packages/compass-indexes/src/components/regular-indexes-table/regular-indexes-table.tsx

```diff
@@ -80,12 +121,14 @@ function sortFn(
     return propSort;
   }
 
-  if (field === 'usage') {
-    field = 'usageCount';
-  }
+  const prop = field as unknown as SortableField;
```

Comment:
Nit: if you turn the check at the top of this function into a method that asserts, you won't need to assert types manually here

```ts
function isSupportedSortField(
  field: string
): field is 'name' | 'type' | 'size' | 'usageCount' | 'properties' {
  return ['name', 'type', 'size', 'usageCount', 'properties'].includes(field);
}

// ...

  if (!isSupportedSortField(field)) {
    return 0;
  }
```